questionnaires/vaping_weekly_survey/vaping_weekly_survey_armt.json

```diff
@@ -0,0 +1,856 @@
+        "form_name": "vaping_weekly_survey",
+        "section_header": "",
+        "field_type": "range",
+        "field_label": "For each item below, please mark the response that best describes you currently.",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "0 I have lots of energy "
+            },
+            {
+                "code": "2",
+                "label": "2 "
+            },
+            {
+                "code": "3",
+                "label": "3 "
+            },
+            {
+                "code": "4",
+                "label": "4 "
+            },
+            {
+                "code": "5",
+                "label": "5 I have no energy at all"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "energy_matrix",
+        "matrix_ranking": "",
+        "field_annotation": "range-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "smoke_days",
+        "form_name": "vaping_weekly_survey",
+        "section_header": "",
+        "field_type": "radio",
+        "field_label": "In the past 7 days, on how many days did you smoke cigarettes?",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "0 "
+            },
+            {
+                "code": "1",
+                "label": "1 "
+            },
+            {
+                "code": "2",
+                "label": "2 "
+            },
+            {
+                "code": "3",
+                "label": "3 "
+            },
+            {
+                "code": "4",
+                "label": "4 "
+            },
+            {
+                "code": "5",
+                "label": "5 "
+            },
+            {
+                "code": "6",
+                "label": "6 "
+            },
+            {
+                "code": "7",
+                "label": "7 "
+            },
+            {
+                "code": "8",
+                "label": "Don't know"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "smoke_days_3",
+        "form_name": "vaping_weekly_survey",
+        "section_header": "",
+        "field_type": "radio",
```

Comment:
can use yes/no type

Reply:
Sorry, new to reading this code, not sure what this refers to.

Reply:
It's just the field type since we have a `yesno` type for radio buttons with only Yes and No choices, and a general `radio` type. So I've just changed this to a `yesno` type since the choices needed are only yes and no.

Reply:
great, thank you